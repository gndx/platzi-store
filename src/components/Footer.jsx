import React from 'react';
import '../styles/components/Footer.styl';

const Footer = () => (
  <div className="Footer">
    <p className="Footer-title">Platzi Store</p>
    <p className="Footer-copy">Todos los Izquierdos Reservados</p>
  </div>
);

export default Footer;
