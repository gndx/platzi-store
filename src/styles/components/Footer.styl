.Footer
  display flex
  justify-content space-between