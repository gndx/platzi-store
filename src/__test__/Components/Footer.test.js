import 'jsdom-global/register';
import React from 'react';
import { mount } from 'enzyme';
import { create } from 'react-test-renderer';
import Footer from '../../components/Footer';

describe('<Footer/>', () => {
  const footer = mount(<Footer />);
  test('Render del componente Footer', () => {
    expect(footer.length).toEqual(1);
  });

  test('Render del titulo', () => {
    expect(footer.find('.Footer-title').text()).toEqual('Platzi Store');
  });
});

describe('Footer Snapshot', () => {
  test('Comprobar la UI del componente', () => {
    const footer = create(<Footer />);
    expect(footer.toJSON()).toMatchSnapshot();
  });
});
