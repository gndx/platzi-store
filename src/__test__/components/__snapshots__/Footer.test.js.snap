// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Snapshot Comprobar la UI del componente Footer 1`] = `
<div
  className="Footer class"
>
  <p
    className="Footer-title"
  >
    Platzi Store
  </p>
  <p
    className="Footer-copy"
  >
    Todos los Izquierdos Reservados
  </p>
</div>
`;
